### PR TITLE
Cult floors now reduce rune scribing for small runes by 50%

### PR DESCRIPTION
**Pull request by Robustin**
Been meaning to give these silly floors a purpose.

### CODE REVIEW

**Comment by juanototo2** [comment]
also make it that either they're buffed on cult floors the runes, get a special effect, or simply are more efficient with blood or something.. if you can get joan/kor/oranges to not screech at you the whole way through.

**Comment by Robustin** [comment]
I am considering trying something along those lines for creating runes on cult walls, this is a pretty unoffensive way to give these floors a purpose besides theming.

**Comment by FantasticFwoosh** [comment]
Makes sense, put down a pylon which is basically a huge screeching "CULT BASE HERE, LOOK AT THE FORCEFULLY CONVERTED FLOORS MADE THROUGH WALLS LEADING INTO MAINT" a little less painful to do when you can speed-scribe a rune in a pinch.

File: code/game/gamemodes/cult/rune_spawn_action.dm [hunk 1]
```diff
@@ -46,6 +46,8 @@
 	if(damage_interrupt && isliving(owner))
 		var/mob/living/L = owner
 		health = list("health" = L.health)
+	if(istype(get_turf(owner), /turf/open/floor/engine/cult))
+		scribe_time = scribe_time * 0.5
```
Comment (ChangelingRain) [inline code/game/gamemodes/cult/rune_spawn_action.dm]: you also need to do this in ritual.dm since this is just for the action

Comment (AnturK) [inline code/game/gamemodes/cult/rune_spawn_action.dm]: Also this will stack, and persist won't it.

Comment (RemieRichards) [inline code/game/gamemodes/cult/rune_spawn_action.dm]: it will, just needs copying into a local variable and using that variable in the ```do_after()```

**Pull request merged**